rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.71.1"
+components = [ "cargo", "clippy", "rustc", "rust-docs", "rust-std" ]
```

Comment:
no rustfmt?

Reply:
No, because we rely on nightly for `rustfmt` 😄

Reply:
Let me add a comment about that...